### PR TITLE
[2.070][ndslice] update win*.mak & uncomment unittest with dummyranges

### PR DESCRIPTION
**Pull request by 9il**
@MartinNowak this PR upgrades the win*.mak files to build docs on windows and uncomments a unittest with dummyranges. The unittets was commented before #3806 was merged.

### CODE REVIEW

File: std/experimental/ndslice/slice.d [hunk 1]
```diff
@@ -2367,9 +2382,7 @@ private auto ptrShell(Range)(Range range, sizediff_t shift = 0)
     return PtrShell!Range(shift, range);
 }
 
-version(none) // TODO: Remove before merge
-// @safe pure nothrow ?
-unittest
```
Comment (9il) [inline std/experimental/ndslice/slice.d]: #3806

**Comment by MartinNowak** [comment]
Sorry there is now a merge conflict b/c I merged back some missing changes from the stable branch.

**Comment by 9il** [comment]
Rebased

**Comment by 9il** [comment]
Rebased again.

I have found a bug for allocators on windows:
```
std\experimental\allocator\mallocator.d(277): Error: @nogc function 'std.experimental.allocator.mallocator.AlignedMallocator.alignedReallocate' cannot call non-@nogc function 'std.experimental.allocator.mallocator.AlignedMallocator.deallocate'
```

I will report it.

**Comment by 9il** [comment]
Yes, https://issues.dlang.org/show_bug.cgi?id=15508

**Comment by 9il** [comment]
I will check it again.

**Comment by 9il** [comment]
But workaround is removed https://github.com/D-Programming-Language/phobos/commit/4cf352c19db9fff49dc7dbbf3ea763c648735748

**Comment by 9il** [comment]
Fixed, it can be merged now.

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

**Comment by 9il** [comment]
Thanks! Should I open the same PR for stable branch or it can be merged to stable another way?

**Comment by 9il** [comment]
The same PR for stable #3905

**Pull request merged**